scopes/compositions/compositions/compositions.ui.runtime.tsx

```diff
@@ -40,10 +42,15 @@ export class CompositionsUI {
       { menuBarWidgetSlot: compositions.menuBarWidgetSlot },
       emptyStateSlot
     );
+    const compositionCompare = new CompositionCompareSection(emptyStateSlot);
 
     component.registerRoute(section.route);
     component.registerNavigation(section.navigationLink, section.order);
-
+    componentCompare.registerNavigation({
+      props: { ...compositionCompare.navigationLink },
+      order: compositionCompare.navigationLink.order,
+    });
+    componentCompare.registerRoutes([compositionCompare.route]);
```

Comment:
if it grows, or any of the others, split into 2 aspects (e.g. compositions aspect and the compositions.compaere aspect